clicommand/pipeline_upload.go

```diff
@@ -401,7 +402,11 @@ func (cfg *PipelineUploadConfig) parseAndInterpolate(src string, input io.Reader
 			}
 			result.Env.Set(tracetools.EnvVarTraceContextKey, tracing)
 		}
-		if err := result.Interpolate(environ); err != nil {
+		preferRuntimeEnv := false
+		if experiments.IsEnabled(ctx, experiments.InterpolationPrefersRuntimeEnv) {
+			preferRuntimeEnv = true
+		}
```

Comment:
This ought to be equivalent:

```suggestion
		preferRuntimeEnv := experiments.IsEnabled(ctx, experiments.InterpolationPrefersRuntimeEnv)
```